app/views/admin/statistics_announcements/_search_results.html.erb

```diff
@@ -0,0 +1,59 @@
+<p class="govuk-heading-s govuk-!-margin-bottom-3"><%= pluralize(number_with_delimiter(paginator.total_count), "document") %></p>
+
+<% if paginator.blank? %>
+  <p class="govuk-body app-view-edition-search-results__no_documents">No <%= filter.options[:dates] %> statistics announcements found</p>
+<% else %>
+  <div class="govuk-table--with-actions">
+    <%= render "govuk_publishing_components/components/table", {
+      head: [
+        {
+          text: "Title"
+        },
+        {
+          text: "Organisation"
+        },
+        {
+          text: "Release date"
+        },
+        {
+          text: "Announcement"
+        },
+        {
+          text: tag.span("View", class: "govuk-visually-hidden")
+        }
+      ],
+      rows:
+        paginator.map do |statistics_announcement|
+          [
+             {
+               text: tag.strong(statistics_announcement.title)
+             },
+             {
+               text: statistics_announcement.organisations.map(&:name).to_sentence
+             },
+             {
+               text: sanitize("#{statistics_announcement.release_date.to_date.strftime("%e&nbsp;%B&nbsp;%Y").html_safe}
+                              <br>
+                              #{statistics_announcement.release_date.strftime("%l:%M%P")}
+                              (#{statistics_announcement.confirmed? ? 'confirmed' : 'provisional'})"
+               ),
+             },
+             {
+               text:  if publication = statistics_announcement.publication
+                        sanitize("#{link_to(publication.title, [:admin, publication], class: "govuk-link")}
+                                  <br>
+                                  (#{publication.current_state})")
+                      else
+                        ""
+                      end
+             },
+             {
+               text: link_to(sanitize("View #{tag.span("View", class: 'govuk-visually-hidden')}"), [:admin, statistics_announcement], class: "govuk-link"),
```

Comment:
The link here should be `View #{tag.span(statistics_announcement.title, class: 'govuk-visually-hidden')}` so that a screen reader can navigate through the links with some context of what they would go to if the link is clicked.